orchestra/proc-macro/src/parse/parse_orchestra_struct.rs

```diff
@@ -393,8 +405,17 @@ impl Parse for SubSystemAttrItems {
 		let wip = extract_variant!(unique, Wip; default = false);
 		let message_capacity = extract_variant!(unique, MessageChannelCapacity take );
 		let signal_capacity = extract_variant!(unique, SignalChannelCapacity take );
+		let with_priority_channel = extract_variant!(unique, WithPriorityChannel; default = false);
 
-		Ok(Self { blocking, wip, sends, consumes, message_capacity, signal_capacity })
+		Ok(Self {
+			blocking,
+			wip,
+			sends,
+			consumes,
+			message_capacity,
+			signal_capacity,
+			with_priority_channel,
```

Comment:
Technically a single subsystem doesn't have a priority channel, but a combination of two subsystems has a priority channel. We need to be very clear to convey this to the user. Here I am not certain which direction the channel is pointing. I presume the `rx.await` will have one and every other subsystem can send a priority message to this one? (without context, just from this particular ident).

Reply:
Given the previous comment, I'd rename to `can_send_priority_msg` (not great) or similar

Reply:
Yes. Was just thinking about it. But it works in the opposite way: ‘can_receive_priority_messages’. My comments about the flag are wrong. The name is long, but it’s better to be crystal clear. As shown in the test, it’s enough to set only receiving subsystem with the flag.